deepmd/pt/model/descriptor/dpa2.py

```
@@ -715,6 +715,7 @@ def forward(
         nlist: torch.Tensor,
         mapping: torch.Tensor | None = None,
         comm_dict: dict[str, torch.Tensor] | None = None,
+        fparam: torch.Tensor | None = None,
```

Ruff (0.15.2)

[warning] Unused method argument: `fparam`

(ARG002)